sqlglot/lineage.py

```diff
@@ -113,7 +113,7 @@ def to_node(
                 )
             else:
                 if table not in tables:
-                    tables[table] = Node(name=table, source=source, expression=source)
+                    tables[table] = Node(name=str(c.this), source=source, expression=source)
```

Comment:
```suggestion
                    tables[table] = Node(name=c.name, source=source, expression=source)
```

Reply:
@GeorgeSittas Just made another change to refer to the fully qualified column name so that the Node's name makes sense in context of its other metadata like `node.downstream` & node.expression` which is referencing the Table node

![image](https://user-images.githubusercontent.com/59633236/219848890-bfe2d1ab-2034-4de9-944c-4fe56994585b.png)

Reply:
Here's a sample of the node's dataclass attributes
```python
Node(name='table.col1', expression=(TABLE this: 
  (IDENTIFIER this: table, quoted: False), db: 
  (IDENTIFIER this: db, quoted: False), catalog: 
  (IDENTIFIER this: catalog, quoted: False), alias: 
  (TABLEALIAS this: 
    (IDENTIFIER this: table, quoted: False))), source=(TABLE this: 
  (IDENTIFIER this: table, quoted: False), db: 
  (IDENTIFIER this: db, quoted: False), catalog: 
  (IDENTIFIER this: catalog, quoted: False), alias: 
  (TABLEALIAS this: 
    (IDENTIFIER this: table, quoted: False))), downstream=[])
```

Reply:
Yea, it makes sense to use the fully qualified name, otherwise we might lose context.